### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by flaviohr**
Some key words were not fully translated because they are commonly used and recognizable by people who have at least minimal exposure to terms related to artificial intelligence and stable diffusion, like 'steps', CFG, etc.

The term ‘Prompt’ remains unchanged because it is widely used in various contexts, and practically everyone is accustomed to hearing it. Additionally, there is no official translation for it. However, this may be subject to change if necessary."

### CODE REVIEW

File: ai_diffusion/language/es.json [hunk 1]
```diff
@@ -0,0 +1,376 @@
+{
+  "id": "es",
+  "name": "español",
```
Comment (Acly) [inline ai_diffusion/language/es.json:3]: Please change first letter to upper case

File: ai_diffusion/language/es.json [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+    "Too many control layers of type": "Demasiadas capas de control del tipo",
+    "Total generated:": "Total generado",
+    "Translate text prompts from the selected language to English": "Traduce los textos del Prompt del idioma seleccionado al inglés.",
+    "Translation is disabled. Click to enable prompt translation from your language to English": "La traducción está desactivada. Haz clic para habilitar la traducción de indicaciones de tu idioma al inglés.",
+    "URL used to connect to a running ComfyUI server. Default is 127.0.0.1:8188 (local).": "URL utilizada para conectarse a un servidor ComfyUI en ejecución. El valor predeterminado es 127.0.0.1:8188 (local).",
+    "Unblur": "Enfoque",
+    "Unblur - Medium": "Enfoque - Medio",
+    "Unblur - Strong": "Enfoque - Fuerte",
+    "Universal": "Universal",
+    "Upgrade": "null",
```
Comment (Acly) [inline ai_diffusion/language/es.json:350]: this looks unintended

File: ai_diffusion/language/es.json (hunk 1 above)
Comment (flaviohr) [reply ai_diffusion/language/es.json]: I'm sorry about that. that's ok now.

File: ai_diffusion/language/es.json (hunk 2 above)
Comment (flaviohr) [reply ai_diffusion/language/es.json]: fixed

**Comment by Acly** [comment]
Added, thank you!

**Pull request merged**